svc-aggregation/system/addplugin.go

```diff
@@ -177,20 +177,21 @@ func (e *ExternalInterface) addPluginData(req AddResourceRequest, taskID, target
 
 		managersData[pluginContactRequest.OID] = body
 	}
-	e.SubscribeToEMB(plugin.ID, queueList)
 	// saving all plugin manager data
 	var listMembers = make([]agresponse.ListMember, 0)
 	for oid, data := range managersData {
-		dbErr := agmodel.GenericSave(updateManagerName(data, plugin.ID), "Managers", oid)
-		if err != nil {
-			errMsg := "error: while saving the plugin data with generic save: " + dbErr.Error()
+
+		dbErr := agmodel.SavePluginManagerInfo(updateManagerName(data, plugin.ID), "Managers", oid)
+		if dbErr != nil {
+			errMsg := "Unable to save the plugin data with SavePluginManagerInfo: " + dbErr.Error()
```

Comment:
dbErr already has this info. Why are we adding this again ?

Reply:
addressed